<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupbox_images_size">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>754</width>
     <height>66</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="title">
    <string>Images size</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <layout class="QHBoxLayout" name="horizontal_layout_14">
      <item>
       <widget class="QLabel" name="image_size">
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Images size (px):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_edit_image_size">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>4096</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontal_spacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="Start_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>782</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="ratio_groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>754</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Ratio of test/train/val</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontal_layout_16">
      <item>
       <widget class="QLabel" name="Test_label">
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Test (%):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Test_percentages">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string>20</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Train_label">
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Train (%):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Train_percentages">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string>70</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="vertical_spacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>5</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontal_layout_17">
      <item>
       <widget class="QLabel" name="Val_label">
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Val (%):&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Val_percentages">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>247</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>10</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontal_spacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>388</width>
          <height>21</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupbox_result_path">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>754</width>
     <height>66</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="title">
    <string>Result Path</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <layout class="QHBoxLayout" name="horizontal_layout_15">
      <item>
       <widget class="QLineEdit" name="Path">
        <property name="text">
         <string>C:\Users\mixli\OneDrive\Рабочий стол\results</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupbox_shapes">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>754</width>
     <height>66</height>
    </rect>
   </property>
   <property name="title">
    <string>Shapes</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontal_layout">
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>124</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>124</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Layer:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="combobox_group">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
